acceptance/setup/pre_suite/50_install_modules.rb

```diff
@@ -4,4 +4,8 @@
 
 step "Install the puppetdb module and dependencies" do
   on master, "puppet module install puppetlabs/puppetdb"
+  if options[:type] == 'aio'
+    # patch puppetdb module for version 4.1.0
```

Comment:
Umm, we do module fixes ourselves, not the module team FWIW @johnduarte, did you want to move that ticket into our queue? We'll need to prioritize it potentially.